EasyPostTest/ShipmentTest.cs

```diff
@@ -166,5 +182,39 @@ public class ShipmentTest {
             rate = shipment.LowestRate(includeCarriers: new List<string>() { "FedEx" });
             Assert.IsNull(rate);
         }
+
+        [TestMethod]
+        public void TestCarrierAccounts()
+        {
+            Address to = Address.Create(toAddress);
+            Address from = Address.Create(fromAddress);
+            Parcel parcel = Parcel.Create(new Dictionary<string, object>() {
+                {"length", 8}, {"width", 6}, {"height", 5}, {"weight", 10}
+            });
+            CustomsItem item = new CustomsItem() { description = "description" };
+            CustomsInfo info = new CustomsInfo()
+            {
+                customs_certify = "TRUE",
+                eel_pfc = "NOEEI 30.37(a)",
+                customs_items = new List<CustomsItem>() { item }
+            };
+
+            Shipment shipment = new Shipment();
+            shipment.to_address = to;
+            shipment.from_address = from;
+            shipment.parcel = parcel;
+            shipment.carrier_accounts = new List<CarrierAccount> { new CarrierAccount { id = "ca_qn6QC6fd" } };
+            shipment.Create();
+            var valid = true;
+            if (shipment.rates.Count > 0)
+            {
+                foreach (var rate in shipment.rates)
+                {
+                    if (rate.carrier_account_id != "ca_qn6QC6fd") valid = false;
+                }
+            }
+
+            Assert.IsTrue(valid);
```

Comment:
Should be able to use [`List<T>.TrueForAll`](https://msdn.microsoft.com/en-us/library/kdxe4x4w%28v=vs.110%29.aspx) here. Everything else looks fine, but can you change your project settings to put the curly brace on the same line as the method definitions and reformat these files?